examples/components/MNIST/traininsilo/run.py

```diff
@@ -229,7 +229,7 @@ def local_train(self, checkpoint):
             test_acc = 0.0
 
             for epoch in range(self._epochs):
-
+                self.model_.train()
```

Comment:
why there is a duplicated self.model_.train() here?
<img width="308" alt="image" src="https://user-images.githubusercontent.com/57924627/215230579-af8229b7-5ddc-4419-99a7-cf2f3b5efb92.png">

Reply:
nice catch! fixed